spec/services/export/spending_breakdown_spec.rb

```diff
@@ -7,13 +7,62 @@
     @activity = create(:project_activity, organisation: @organisation)
     @source_fund = Fund.new(1)
 
-    create_q1_2020_actual_and_adjustments
-    create_q4_2020_actual_and_adjustments
+    q1_2019_report = create(
+      :report,
+      :approved,
+      organisation: @organisation,
+      fund: @activity.associated_fund,
+      financial_quarter: 1,
+      financial_year: 2019
+    )
 
-    create_q1_2020_refund_and_adjustments
-    create_q4_2020_refund_and_adjustments
+    ForecastHistory
+      .new(@activity, report: q1_2019_report, financial_quarter: 1, financial_year: 2020)
+      .set_value(10_000)
+    ForecastHistory
+      .new(@activity, report: q1_2019_report, financial_quarter: 4, financial_year: 2020)
+      .set_value(5_00)
+    ForecastHistory
+      .new(@activity, report: q1_2019_report, financial_quarter: 1, financial_year: 2021)
+      .set_value(10_000)
+    ForecastHistory
+      .new(@activity, report: q1_2019_report, financial_quarter: 4, financial_year: 2021)
+      .set_value(20_000)
 
-    create_old_report_and_forecasts
+    q4_2019_report = create(
+      :report,
+      :approved,
+      organisation: @organisation,
+      fund: @activity.associated_fund,
+      financial_quarter: 4,
+      financial_year: 2019
+    )
+
+    ForecastHistory
+      .new(@activity, report: q4_2019_report, financial_quarter: 1, financial_year: 2020)
+      .set_value(5_000)
+    ForecastHistory
+      .new(@activity, report: q4_2019_report, financial_quarter: 4, financial_year: 2020)
+      .set_value(2_500)
+    ForecastHistory
+      .new(@activity, report: q4_2019_report, financial_quarter: 1, financial_year: 2021)
+      .set_value(20_000)
+    ForecastHistory
+      .new(@activity, report: q4_2019_report, financial_quarter: 4, financial_year: 2021)
+      .set_value(10_000)
+
+    @q1_report = create(:report, financial_quarter: 1, financial_year: 2020)
+    @q2_report = create(:report, financial_quarter: 2, financial_year: 2020)
```

Comment:
do you think it would be clearer to present these 8 forecast fixtures as a table also? e.g.

```
    create_forecast_fixtures(
      <<~TABLE
        |report|financial_quarter|financial_year|value|
        |q1_19 |   1             |   2020       |10000|
        |q1_19 |   4             |   2020       | 5000|
        |q1_19 |   1             |   2021       |10000|
        |q1_19 |   4             |   2021       |20000|
        |q4_19 |   1             |   2020       | 5000|
        |q4_19 |   4             |   2020       | 2500|
        |q4_19 |   1             |   2021       |20000|
        |q4_19 |   4             |   2021       |10000|
      TABLE
    )
```

Reply:
Can you wait? It is coming after the forecast work in #1437 is used in the spending breakdown?

Reply:
yes, of course! I saw that you are using tables of forecast fixtures in the next PR!